packages/client-sdk-web/src/internal/pubsub-client.ts

```diff
@@ -36,8 +36,8 @@ export class PubsubClient<
   protected readonly logger: MomentoLogger;
   private readonly clientMetadataProvider: ClientMetadataProvider;
 
-  private static readonly RST_STREAM_NO_ERROR_MESSAGE =
-    'Received RST_STREAM with code 0';
```

Comment:
Does the deleted message not happen in the web client?

Reply:
hmm good point, I didn't see this in my testing, but you are right I think this could still happen if there is a deployment going on. I will update it to retry on both of these errors